src/main/webapp/resources/js/components/tabs/SideTabs.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Component to render styled side tabs. Use
+ * this whenever vertical tabs within a page
+ * are required
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { Tabs } from "antd";
+import styled from "styled-components";
+
+const VerticalTabs = styled(Tabs)`
+  .ant-tabs-tab {
+    @media only screen and (min-width: 800px) {
+      width: 200px;
+    }
+  }
+`;
+
+export function SideTabs(props) {
```

Comment:
Here you could set the defaults in the method declaration:

```js
export function SideTabs({
  type = "card",
  tabPosition = "left",
  animated = false,
  activeKey,
  onChange,
  children,
  ...props
}) {
  return (
    <VerticalTabs
      type={type}
      activeKey={activeKey}
      onChange={onChange}
      tabPosition={tabPosition}
      animated={animated}
      {...props}
    >
      {children}
    </VerticalTabs>
  );
}
```

Reply:
Fixed in 5e42840cf8b9aa72a4ab4384c86a925e103d0382